cilium-health status -d causes segfault

If you manually run `cilium-health` with the wrong arguments like this then it fails:

```
root@minikube:~# cilium-health status -d
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x94c293]

goroutine 1 [running]:
github.com/cilium/cilium/cilium-health/cmd.glob..func3(0xe69080, 0xc42041fb00, 0x0, 0x1)
        /go/src/github.com/cilium/cilium/cilium-health/cmd/status.go:49 +0x1c3
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).execute(0xe69080, 0xc42041faf0, 0x1, 0x1, 0xe69080, 0xc42041faf0)
        /go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:648 +0x234
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xe696e0, 0x0, 0xc42012e500, 0xac0b82)
        /go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:735 +0x2d4
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).Execute(0xe696e0, 0xc42007e058, 0x0)
        /go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:693 +0x2b
github.com/cilium/cilium/cilium-health/cmd.Execute()
        /go/src/github.com/cilium/cilium/cilium-health/cmd/root.go:69 +0x2d
main.main()
        /go/src/github.com/cilium/cilium/cilium-health/main.go:20 +0x20
```

This should complain about invalid combination of arguments or something instead.

--- a/cilium-health/cmd/status.go
+++ b/cilium-health/cmd/status.go
@@ -39,6 +39,10 @@ var statusGetCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		var sr *models.HealthStatusResponse
 
+		if client == nil {
+			Fatalf("Invalid combination of arguments")
+		}
+
 		if probe {
 			result, err := client.Connectivity.PutStatusProbe(nil)
 			if err != nil {

